### ISSUE TITLE
Emphasis style error not detected

### ISSUE THREAD

**Issue by AlbertoFabbri93** [rvben/rumdl#715]
### rumdl version

0.2.30

### Installation method

Homebrew

### Bug description

When a file is using a emphasis style consistently no error is thrown even when e.g. the emphasis style is set to `style = "asterisk"` but the file is using underscores.

### Steps to reproduce

1. Set `.rumdl.toml` to:

```toml
# Ensures all italic text uses the same style throughout your document
[emphasis-style]
# Always use *text* for italics
style = "asterisk"
```
 
2. Create a file `test.md` with the following content:

```
# Test

## rumdl formatter

- This is a __test__.
```

3. Run `rumdl check`

### Expected behavior

An `emphasis-style` error is reported.

### Actual behavior

The command returns `No issues found`.

### Configuration

```toml

```

### Operating system

macOS

**Comment by rvben** [comment]
Thanks for the clear repro. This is working as intended, though the rule split is easy to miss: `__test__` is strong emphasis (bold), which is governed by `strong-style` (MD050), while `emphasis-style` (MD049) covers single-marker emphasis (italics), and your test file contains no italics. With `[strong-style] style = "asterisk"` rumdl flags your example (`MD050: Strong emphasis should use ** instead of __`), and you can set both rules together for full consistency. I verified all three combinations against your exact config and file on 0.2.30.

**Comment by AlbertoFabbri93** [comment]
Thank you very much for the help, silly mistake. I even added the comments to explain that particular rule but apparently I didn't pay enough attention. Anyway, the rules do indeed work as intended, thanks for a great software!
